core/frontend/src/IModelConnection.ts

```diff
@@ -85,12 +85,12 @@ export abstract class IModelConnection extends IModel {
    * @internal
    */
   public readonly geoServices: GeoServices;
-  /** @internal Whether it has already been determined that this iModelConnection does not have a map projection. */
-  protected _noGcsDefined?: boolean;
-  /** @internal Whether it has already been determined that this iModelConnection does not have a map projection. */
-  public get noGcsDefined(): boolean | undefined { return this._noGcsDefined; }
+  /** @internal Whether GCS has been disabled for this iModelConnection. */
+  protected _gcsDisabled?: boolean;

```

Comment:
Should declare as true | undefined, since you don't check/allow false...

Reply:
Or just a boolean that defaults to false.

Reply:
changed this.

Reply:
@bbastings - resolve this comment if looks good?

Reply:
Why is noGCSDefined returning boolean | undefined? Would b cleaner just to make disableGCS a boolean that defaults to false and have noGCSDefined return true or false...

Reply:
@bbastings Good point, thanks. The boolean | undefined was a holdover from the way the code used to be.  I just pushed a change to make it return boolean, as well as making _gcsDisabled a boolean defaulting to false.